### ISSUE TITLE
macropad_simpletest_display.py fails in adafruit_display_text/bitmap_label.py

### ISSUE THREAD

**Issue by tlyu** [adafruit/Adafruit_CircuitPython_MacroPad#53]
While trying to run `macropad_simpletest_display.py` from `adafruit-circuitpython-bundle-9.x-mpy-20240118`, i get the following traceback:

```
Traceback (most recent call last):
  File "code.py", line 13, in <module>
  File "adafruit_macropad.py", line 937, in display_text
  File "adafruit_simple_text_display.py", line 165, in __init__
  File "adafruit_display_text/bitmap_label.py", line 118, in __init__
  File "adafruit_display_text/bitmap_label.py", line 214, in _reset_text
  File "adafruit_display_text/bitmap_label.py", line 455, in _place_text
  File "adafruit_display_text/bitmap_label.py", line 492, in _blit
ValueError: x1 must be 0-77
```

I recall that this example code worked with 8.x.

It looks like something deep in bitmaptools is doing range validation incorrectly.

```
Adafruit CircuitPython 9.0.0-alpha.6-28-g150db00eb3-dirty on 2024-01-18; Adafruit Macropad RP2040 with rp2040
```

**Comment by tlyu** [comment]
Confirmed that adafruit/circuitpython#8823 appears to fix this issue. Thanks to @FoamyGuy for pointing me at it via Discord!

**Comment by tlyu** [comment]
Fixed by adafruit/circuitpython#8823.
